feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/widget/TimeTableLayoutManager.kt

```diff
@@ -55,7 +55,7 @@ class TimeTableLayoutManager(
         calculateColumns()
 
         var xOffset = parentLeft
-        for (columnNumber in 0 until columns.size) {
+        for (columnNumber in columns.keys.min()!!..columns.keys.max()!!) {
```

Comment:
Could you please use SparseArray instead of HashMap?
`SparseArray#keyAt(0)` will return the minimum key and `SparseArray#keyAt(size - 1)` can return the maximum key with smaller computational complexity and memory usage.